extensions/sequence.md

```diff
@@ -0,0 +1,51 @@
+# Sequence
+
+This extension defines two attributes that can be included within a CloudEvent
+to describe the position of the event relative to the other events from the
+same event source.
+
+The `sequence` attribute represents the value of this event's
+order in the stream of events.  The exact value and meaning of this
+attribute is defined by the `sequenceType` attribute. 
+If the `sequenceType` is missing, or not defined in this specification,
+event consumers will need to have some out-of-band communication with the
+event producer to understand how to interpret the value of the attribute.
+
+## Attributes
+
+### sequence
+* Type: `String`
+* Description: Value expressing the relative order of the event. This enables
+  interpretation of data supercedence.
+* Constraints
+  * REQUIRED
+  * MUST be a non-empty lexicographically-orderable string
+  * RECOMMENDED as contiguous
```

Comment:
Possibly:
`MUST be monotonically increasing`

Reply:
at least monotonic

Reply:
added "monotonically". Wasn't sure about the "increasing" part since that might imply a numeric based sequence and I'm assuming there might be some that don't fit that model. Is that a correct assumption?

Reply:
Answering because you asked.  You've already added "increasing".

I feel a little uncertain and would defer to a mathematician but the `lexicographically orderable` requirement gives an absolute ordering across all unique values.  I would expect "increasing" to make sense in that context and mean "each subsequent value would be ordered/sorted after all preceding values".

For the record, my initial question would disallow the reasonable circular counter described in the `Integer` sequence type and therefore could not be a MUST.